mypy.ini

```diff
@@ -31,4 +31,4 @@ ignore_missing_imports = True
 ignore_missing_imports = True
 
 [mypy-pandas.*]
-ignore_missing_imports = True
+ignore_missing_imports = True
```

Comment:
add back new line? unless this style is preferred by an auto-formatter, let me know if so